modules/virt-vm-storage-volume-types.adoc

```diff
@@ -44,9 +45,7 @@ disk.
 |containerDisk
 |References an image, such as a virtual machine disk, that is stored in
 the container image registry. The image is pulled from the registry and
-embedded in a volume when the virtual machine is created. A
-*containerDisk* volume is ephemeral. It is discarded when
-the virtual machine is stopped, restarted, or deleted.
+embedded in a volume when the virtual machine is created.
```

Comment:
Stroke the sentence, instead:
`The image is pulled from the registry and attached as a disk to the virtual machine when it is getting launched.`